index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>ProgramBoys</title>
+</head>
+<body>
+    <div class="container">
+        <div class="caixaTexto1">
+            <div class="titulo">
+                <p>Primeira página do projeto</p>
+            </div>
+            <div class="integrantes">
+                <div class="participante">
+                    <p>Pedro Britto</p>
+                </div>
+                <div class="participante">
+                    <p>Tiago Abreu</p>
```

Comment:
Não gostei desta opcão
```suggestion
                    <p>Pedrinho</p>
```